### PR TITLE
Makefile

### PR DESCRIPTION
**Pull request by cpanato**
Download jira 1.0.1 for release 5.6

### CODE REVIEW

File: build/release.mk [hunk 1]
```diff
@@ -72,39 +73,66 @@ endif
...
+	@for plugin_package_legacy in $(PLUGIN_PACKAGES_LEGACY) ; do \
+		curl -s https://api.github.com/repos/mattermost/$$plugin_package_legacy | grep browser_download_url | grep darwin | cut -d '"' -f 4 | wget -qi - -P  $(DIST_PATH)/prepackaged_plugins/ ;\
+	done
 endif
 	@# Package
 	tar -C dist -czf $(DIST_PATH)-$(BUILD_TYPE_NAME)-osx-amd64.tar.gz mattermost
 	@# Cleanup
 	rm -f $(DIST_PATH)/bin/mattermost
 	rm -f $(DIST_PATH)/bin/platform
+	rm -f $(DIST_PATH)/prepackaged_plugins/*jira*
```
Comment (jwilander) [inline build/release.mk]: What is this part for?

Comment (cpanato) [reply build/release.mk]: to remove the platform specific, this jira release was builded for platforms so we need to remove when we package, then the next platform will get a clean dist

**Comment by jasonblais** [comment]
Will we cherry-pick and/or submit a PR for `master` and `release-5.7` branches as well, once we've verified the fix?

**Comment by cpanato** [comment]
I'm fine to submit to master and cherry pick to 5.7
any thoughts @jwilander

**Comment by jwilander** [comment]
Nah let's just put this in 5.6 and 5.6. We'll fix the jira plugin for master

**Pull request merged**